bundle/Resources/config/routing.yml

```diff
@@ -26,3 +26,10 @@ _ezpublishLegacyRest:
         _controller: ezpublish_legacy.rest.controller:restAction
     requirements:
         path: .*
+
+_ezpublishLegacyLayoutSet:
+    path: /layout/set/{path}
```

Comment:
> does this work with uri SA

Yes, as we override default router tu use semanticPathinfo. So this will always work.
The requirements is just to allow `/` for `{path}`